beacon_node/beacon_chain/src/custody_context.rs

```diff
@@ -249,8 +367,8 @@ impl<E: EthSpec> CustodyContext<E> {
                     "Custody group count updated"
                 );
                 return Some(CustodyCountChanged {
-                    new_custody_group_count: updated_cgc,
-                    sampling_count: self.num_of_custody_groups_to_sample(effective_epoch, spec),
+                    new_custody_group_count: updated_cgc as u64,
+                    sampling_count: self.sampling_count_at_epoch(None, spec) as u64,
```

Comment:
I noticed `num_of_custody_groups_to_sample` and `num_of_data_columns_to_sample` are now both mapped to `sampling_count_at_epoch` - I'm sure this is intentional, but I want to make sure the reason for the previous separation was clear - it was mainly to support subnet decoupling (which isn't my favourite feature) and unifying this to a single function is likely going to break that.